modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/item/action/AnalyticsReportsContentDashboardItemActionProvider.java

```diff
@@ -49,6 +73,10 @@
 		String className, long classPK, HttpServletRequest httpServletRequest,
 		Locale locale) {
 
+		if (!_isShow(className, classPK, httpServletRequest)) {
```

Comment:
Better a public separate method for the permissions? Of course, we can keep calling it here, but better if the isShow is public.

Reply:
what do you mean? add this method to the provider interface?